frontend/src/modules/entitieslist/components/Entity.js

```diff
@@ -123,3 +161,13 @@ export default class Entity extends React.Component<Props> {
         );
     }
 }
+
+
+const mapStateToProps = (state: Object): Props => {
+    return {
+        isReadOnlyEditor: entities.selectors.isReadOnlyEditor(state),
+        isTranslator: user.selectors.isTranslator(state),
+    };
+};
```

Comment:
Why would you connect and not just pass this down from `EntitiesList` like you did for the `batchactions` state?

Reply:
I have no idea what I was thinking. Updated!